meetings/2021-07/july-13.md

```diff
@@ -0,0 +1,512 @@
+
+CP: We also have a polyfill that we have implemented a few months ago, we have test for that too. We actually validate one of Jack's concerns which was whether, or not the callable boundary was enough to be able to implement a membrane on top of it. So we have a very tiny Library, about 1kb, 1.5kb, very simple implementation, but a robust implementation of a membrane library. That allows you to do that kind of advanced use cases if that's what you need. Many of the use cases don't even need that, instead you just need a way to create a realm to evaluate code and maybe get access to some bits out of the realm. So with that in mind, we believe that we're ready for stage 3 and we are ready to open the debate. Questions?
+
+JHD:. So the first one I added was, I mean if this event since this membrane Library, so tiny, why not just support that directly in the language as opposed to making people add this easy to get wrong Library, even if it's small?
```

Comment:
```suggestion
JHD:. So the first one I added was, I mean if this event since this membrane library is so tiny, why not just support that directly in the language as opposed to making people add this easy-to-get-wrong library, even if it's small?
```